CRM/Upgrade/Incremental/php/FiveOne.php

```diff
@@ -61,6 +61,17 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
     // }
   }
 
+  /**
+   * Upgrade function.
+   *
+   * @param string $rev
+   */
+  public function upgrade_5_1_beta1($rev) {
```

Comment:
@totten @eileenmcnaughton should this be `upgrade_5_1_0` instead given the beata may actually not get released?

Reply:
I think so - but would be good to hear from Tim

Reply:
Rule of thumb would be:

* If the patch is against `master`, then use `alpha` (e.g. `upgrade_5_1_alpha1()`).
* If the patch is against the RC branch, then use `beta` (e.g. `upgrade_5_1_beta1()`).
* If the patch is a backport to a stable branch... then please reconsider. Patch releases generally shouldn't need DB changes. But if it's absolutely needed, then use the number of the upcoming release (e.g. `upgrade_5_1_1()` or `upgrade_5_1_2()`).

The best time to change schema is during the `master`/`alpha` phase. Just put any upgrades in the `5.X.alpha1`.

For changes later in the process, it's probably a good idea to be verbal about the target version#. You may need to bump the version# up with `set-version.php`, but the decision would have implications for anyone running the RC/stable. There may be times when it's appropriate to bump to `5.X.beta2` or `5.X.1`. This only comes up if patching an RC or stable branch.

Reply:
Renamed the function to `upgrade_5_1_alpha1()` since this is against the master branch.  Thanks @totten @eileenmcnaughton @seamuslee001